crates/red_knot_python_semantic/resources/mdtest/type_properties/is_subtype_of.md

```diff
@@ -63,52 +81,106 @@ static_assert(is_subtype_of(Never, AlwaysFalsy))
 from typing_extensions import Literal, LiteralString
 from knot_extensions import is_subtype_of, static_assert
 
-static_assert(is_subtype_of(Literal[1], int))
-static_assert(is_subtype_of(Literal[1], object))
-
+# Boolean literals
 static_assert(is_subtype_of(Literal[True], bool))
 static_assert(is_subtype_of(Literal[True], int))
 static_assert(is_subtype_of(Literal[True], object))
 
+# Integer literals
+static_assert(is_subtype_of(Literal[1], int))
+static_assert(is_subtype_of(Literal[1], object))
+
+static_assert(not is_subtype_of(Literal[1], bool))
+
+# See: [special case for float and complex]
```

Comment:
this doesn't link either unfortunately as it's a code block 😢 maybe just paste in the raw URL here?